Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -519,8 +519,9 @@ def _create_hash_filter(hash_arg):
 
 def _create_exposure_filter(exposure_arg):
     # Because the CLI can't accept lists, convert the args to a list if the type is string.
-    if isinstance(exposure_arg, str):
-        exposure_arg = exposure_arg.split(",")
+    exposure_arg = [arg.strip() for arg in exposure_arg.split(",")]
```

Comment:
Handles case when exposure contains space after ','. 

Also had to remove above condition to get the desired behaviour. Couldn't actually debug why condition wasn't getting true, refer the difference in last two commits to see what I am trying to convey if it is not clear.

Reply:
I think this is fine... but keep in mind that the CLI is not the only way to call commands, you can execute them in additional scripts as well, in which case having to pass in a string array might feel odd. It is ok though. Kind of baffling why you couldn't reach the condition... I wonder what the actual "type" is of the argument. Maybe it's like a CLIArgument class or something